<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Однородный центрально-композиционный план</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1051</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QTableWidget" name="table_plan">
       <property name="rowCount">
        <number>79</number>
       </property>
       <property name="columnCount">
        <number>74</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>105</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>25</number>
       </attribute>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <item row="0" column="0">
        <property name="text">
         <string>Опыт</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string>x0</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string>x1</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="0" column="3">
        <property name="text">
         <string>x2</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="0" column="4">
        <property name="text">
         <string>x3</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="0" column="5">
        <property name="text">
         <string>x4</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="0" column="6">
        <property name="text">
         <string>x5</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="0" column="7">
        <property name="text">
         <string>x6</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="0" column="8">
        <property name="text">
         <string>x1*x2</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="0" column="9">
        <property name="text">
         <string>x1*x3</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="0" column="10">
        <property name="text">
         <string>x1*x4</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="0" column="11">
        <property name="text">
         <string>x1*x5</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="0" column="12">
        <property name="text">
         <string>x1*x6</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="0" column="13">
        <property name="text">
         <string>x2*x3</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="0" column="14">
        <property name="text">
         <string>x2*x4</string>
        </property>
       </item>
       <item row="0" column="15">
        <property name="text">
         <string>x2*x5</string>
        </property>
       </item>
       <item row="0" column="16">
        <property name="text">
         <string>x2*x6</string>
        </property>
       </item>
       <item row="0" column="17">
        <property name="text">
         <string>x3*x4</string>
        </property>
       </item>
       <item row="0" column="18">
        <property name="text">
         <string>x3*x5</string>
        </property>
       </item>
       <item row="0" column="19">
        <property name="text">
         <string>x3*x6</string>
        </property>
       </item>
       <item row="0" column="20">
        <property name="text">
         <string>x4*x5</string>
        </property>
       </item>
       <item row="0" column="21">
        <property name="text">
         <string>x4*x6</string>
        </property>
       </item>
       <item row="0" column="22">
        <property name="text">
         <string>x5*x6</string>
        </property>
       </item>
       <item row="0" column="23">
        <property name="text">
         <string>x1*x2*x3</string>
        </property>
       </item>
       <item row="0" column="24">
        <property name="text">
         <string>x1*x2*x4</string>
        </property>
       </item>
       <item row="0" column="25">
        <property name="text">
         <string>x1*x2*x5</string>
        </property>
       </item>
       <item row="0" column="26">
        <property name="text">
         <string>x1*x2*x6</string>
        </property>
       </item>
       <item row="0" column="27">
        <property name="text">
         <string>x1*x3*x4</string>
        </property>
       </item>
       <item row="0" column="28">
        <property name="text">
         <string>x1*x3*x5</string>
        </property>
       </item>
       <item row="0" column="29">
        <property name="text">
         <string>x1*x3*x6</string>
        </property>
       </item>
       <item row="0" column="30">
        <property name="text">
         <string>x1*x4*x5</string>
        </property>
       </item>
       <item row="0" column="31">
        <property name="text">
         <string>x1*x4*x6</string>
        </property>
       </item>
       <item row="0" column="32">
        <property name="text">
         <string>x1*x5*x6</string>
        </property>
       </item>
       <item row="0" column="33">
        <property name="text">
         <string>x2*x3*x4</string>
        </property>
       </item>
       <item row="0" column="34">
        <property name="text">
         <string>x2*x3*x5</string>
        </property>
       </item>
       <item row="0" column="35">
        <property name="text">
         <string>x2*x3*x6</string>
        </property>
       </item>
       <item row="0" column="36">
        <property name="text">
         <string>x2*x4*x5</string>
        </property>
       </item>
       <item row="0" column="37">
        <property name="text">
         <string>x2*x4*x6</string>
        </property>
       </item>
       <item row="0" column="38">
        <property name="text">
         <string>x2*x5*x6</string>
        </property>
       </item>
       <item row="0" column="39">
        <property name="text">
         <string>x3*x4*x5</string>
        </property>
       </item>
       <item row="0" column="40">
        <property name="text">
         <string>x3*x4*x6</string>
        </property>
       </item>
       <item row="0" column="41">
        <property name="text">
         <string>x3*x5*x6</string>
        </property>
       </item>
       <item row="0" column="42">
        <property name="text">
         <string>x4*x5*x6</string>
        </property>
       </item>
       <item row="0" column="43">
        <property name="text">
         <string>x1*x2*x3*x4</string>
        </property>
       </item>
       <item row="0" column="44">
        <property name="text">
         <string>x1*x2*x3*x5</string>
        </property>
       </item>
       <item row="0" column="45">
        <property name="text">
         <string>x1*x2*x3*x6</string>
        </property>
       </item>
       <item row="0" column="46">
        <property name="text">
         <string>x1*x2*x4*x5</string>
        </property>
       </item>
       <item row="0" column="47">
        <property name="text">
         <string>x1*x2*x4*x6</string>
        </property>
       </item>
       <item row="0" column="48">
        <property name="text">
         <string>x1*x2*x5*x6</string>
        </property>
       </item>
       <item row="0" column="49">
        <property name="text">
         <string>x1*x3*x4*x5</string>
        </property>
       </item>
       <item row="0" column="50">
        <property name="text">
         <string>x1*x3*x4*x6</string>
        </property>
       </item>
       <item row="0" column="51">
        <property name="text">
         <string>x1*x3*x5*x6</string>
        </property>
       </item>
       <item row="0" column="52">
        <property name="text">
         <string>x1*x4*x5*x6</string>
        </property>
       </item>
       <item row="0" column="53">
        <property name="text">
         <string>x2*x3*x4*x5</string>
        </property>
       </item>
       <item row="0" column="54">
        <property name="text">
         <string>x2*x3*x4*x6</string>
        </property>
       </item>
       <item row="0" column="55">
        <property name="text">
         <string>x2*x3*x5*x6</string>
        </property>
       </item>
       <item row="0" column="56">
        <property name="text">
         <string>x2*x4*x5*x6</string>
        </property>
       </item>
       <item row="0" column="57">
        <property name="text">
         <string>x3*x4*x5*x6</string>
        </property>
       </item>
       <item row="0" column="58">
        <property name="text">
         <string>x1*x2*x3*x4*x5</string>
        </property>
       </item>
       <item row="0" column="59">
        <property name="text">
         <string>x1*x2*x3*x4*x6</string>
        </property>
       </item>
       <item row="0" column="60">
        <property name="text">
         <string>x1*x2*x3*x5*x6</string>
        </property>
       </item>
       <item row="0" column="61">
        <property name="text">
         <string>x1*x2*x4*x5*x6</string>
        </property>
       </item>
       <item row="0" column="62">
        <property name="text">
         <string>x1*x3*x4*x5*x6</string>
        </property>
       </item>
       <item row="0" column="63">
        <property name="text">
         <string>x2*x3*x4*x5*x6</string>
        </property>
       </item>
       <item row="0" column="64">
        <property name="text">
         <string>x1*x2*x3*x4*x5*x6</string>
        </property>
       </item>
       <item row="0" column="65">
        <property name="text">
         <string>x1^2 - S</string>
        </property>
       </item>
       <item row="0" column="66">
        <property name="text">
         <string>x2^2 - S</string>
        </property>
       </item>
       <item row="0" column="67">
        <property name="text">
         <string>x3^2 - S</string>
        </property>
       </item>
       <item row="0" column="68">
        <property name="text">
         <string>x4^2 - S</string>
        </property>
       </item>
       <item row="0" column="69">
        <property name="text">
         <string>x5^2 - S</string>
        </property>
       </item>
       <item row="0" column="70">
        <property name="text">
         <string>x6^2 - S</string>
        </property>
       </item>
       <item row="0" column="71">
        <property name="text">
         <string>Y</string>
        </property>
       </item>
       <item row="0" column="72">
        <property name="text">
         <string>Yр</string>
        </property>
       </item>
       <item row="0" column="73">
        <property name="text">
         <string>|Y-Yр|</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string/>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="3" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="4" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="5" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="6" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="7" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_do_plan">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>410</y>
      <width>189</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Провести эксперимент</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>410</y>
      <width>189</width>
      <height>70</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="label_a">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_37">
       <property name="text">
        <string>Alpha</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_length">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>195</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>S</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="entry_gen1_int_min">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>410</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>40</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entry_gen1_int_max">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>410</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>60</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entry_gen2_int_min">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>470</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>40</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entry_gen2_int_max">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>470</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>60</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entry_proc1_int_min">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>530</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entry_proc1_int_max">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>530</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>200</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entry_proc1_dev_min">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>440</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entry_proc1_dev_max">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>440</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>200</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entry_proc2_int_min">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>560</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entry_proc2_int_max">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>560</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>200</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entry_proc2_dev_min">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>500</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entry_proc2_dev_max">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>500</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>200</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_34">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>410</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>min интенсивность генератора 1:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_35">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>min интенсивность генератора 2:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_36">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>min интенсивность ОА на 1 типе:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_38">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>min диспесия генератора 1:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_39">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>min интенсивность ОА на 2 типе:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_40">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>min диспесия генератора 2:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_41">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>410</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>max интенсивность генератора 1:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_42">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>470</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>max интенсивность генератора 2:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_43">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>530</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>max интенсивность ОА на 1 типе:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_44">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>440</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>max диспесия генератора 1:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_45">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>560</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>max интенсивность ОА на 2 типе:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_46">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>500</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>max диспесия генератора 2:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_y">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>600</y>
      <width>1031</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
